extensions/hashnode/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hashnode",
+  "title": "Hashnode",
+  "description": "Read, search and create stories from Raycast.",
+  "icon": "command-icon.png",
+  "author": "dpnick",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "featured",
+      "title": "Read Featured",
+      "subtitle": "Hashnode",
+      "description": "read featured posts",
+      "mode": "view"
+    },
+    {
+      "name": "best",
+      "title": "Read Best",
+      "subtitle": "Hashnode",
+      "description": "read best posts",
+      "mode": "view"
+    },
+    {
+      "name": "new",
+      "title": "Read New",
+      "subtitle": "Hashnode",
+      "description": "read new posts",
+      "mode": "view"
+    },
+    {
+      "name": "new_story",
+      "title": "Create Draft Story",
+      "subtitle": "Hashnode",
+      "description": "create a new story",
+      "mode": "view"
```

Comment:
Since the command is just opening a url, you should set the mode to `no-view` and not return react component in `new_story`, the code will be much simpler to follow and you're sure that the screen won't flash!

```js
export default async function NewStory() {
   await open(`https://hashnode.com/create/story?title=${encodeURIComponent("Created from Raycast ⚡️")}`);
 }
```

Reply:
it's a bit hidden in the docs indeed, something we have to improve for sure!